crates/cdk-mint-rpc/src/lib.rs

```diff
@@ -0,0 +1,5 @@
+pub mod proto;
+
+pub mod mint_rpc_cli;
+
+pub use proto::*;
```

Comment:
```suggestion
pub mod mint_rpc_cli;
pub mod proto;
pub use proto::*;
```
NIT: alphabetic sorting